packages/components/src/components/route-with-subroutes/route-with-subroutes.tsx

```diff
@@ -10,6 +11,22 @@ import {
     default_title,
 } from '@deriv/shared';
 
+type TRoute = RouteProps & { default: boolean };
+
+type TRoutesWithSubroutes = {
+    component: React.ElementType | typeof Redirect;
+    exact?: boolean;
+    getTitle: () => void;
+    is_authenticated?: boolean;
```

Comment:
`getTitle` have a fallback in the component. so is this prop required or optional?

Reply:
It should be Optional as per the code